clients/src/main/java/org/apache/kafka/common/metrics/Metrics.java

```diff
@@ -244,6 +269,24 @@ synchronized void registerMetric(KafkaMetric metric) {
         return this.metrics;
     }
 
+    /**
+     * This iterates over every Sensor and triggers a removeSensor if it has expired
+     * Package private for testing
+     */
+    class ExpireSensorTask implements Runnable {
+        public void run() {
+            for (Map.Entry<String, Sensor> sensor : sensors.entrySet()) {
```

Comment:
`sensor` -> `sensorEntry` (since the `getValue` below then looks less confusing)